packages/headless/package.json

```diff
@@ -76,6 +76,8 @@
     "next": "^10.0.3",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.1.2",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
```

Comment:
Were these reintroduced so that the Jest tests for the `src/components/menu/Menu.test.tsx` component pass?

Adding them back as `devDependencies` here causes `lerna bootstrap` to fail at the `npm run build` step. I think that's why @claygriffiths removed them in https://github.com/wpengine/headless-framework/pull/54.

Not sure of the best fix here, though — perhaps we could remove these lines as well as the Menu test for now and test the menu as part of our Cypress tests instead?

Reply:
I think these tests were introduced pre-TypeScript integration in which the react framework packages were swapped out for `@types/react`.

Do the tests need to be rewritten for TypeScript?

Reply:
Adding this did allow the tests to pass.  I don't know much about how package.json devDep work, but added them in and things were 🟢  so I went with it. 😁

Reply:
Just removed Menu.test.jsx and reverted these package json changes.  Will that be sufficient for now?

Reply:
I think it's ok and we can add it back in another issue.

Reply:
I agree, thanks for taking care of that @markkelnar.